### PR TITLE
DataAccess.SQLite fixes

### PR DESCRIPTION
**Pull request by StasDash**
Postgre(pqxx) specific exceptions are ignored now

### CODE REVIEW

File: DataAccess/DataAccess.SQLite/LiteAdapter.hpp [hunk 1]
```diff
@@ -1,74 +1,82 @@
...
-    /** @brief Method for closing connection.
-     *   @details Method for change connection to the database.
+    /**
+     * @brief Method for closing connection.
+     * @details Method for change connection to the database.
      */
-    void closeConnection(void) override;
+    void closeConnection() override;
+
+public:
```
Comment (vDrWool) [inline DataAccess/DataAccess.SQLite/LiteAdapter.hpp:59]: All from line 59 to 82 have to be in private section

File: DataAccess/DataAccess.SQLite/LiteTable.hpp [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
 class LiteTable : public QueryBuilder<QSqlQuery>
 {
 public:
-    LiteTable(const std::string& tableName)
-        : QueryBuilder(DatabaseType::DB_LITE, tableName) 
+    /// Table constructor
+    explicit LiteTable(const std::string& tableName) : QueryBuilder(Utility::DatabaseType::DB_LITE, tableName)
     {
-        _adapter = LiteAdapter::getInstance<LiteAdapter>();
+        _adapter = LiteAdapter::Instance("local.db");
```
Comment (vDrWool) [inline DataAccess/DataAccess.SQLite/LiteTable.hpp:27]: Method Instance is using only inside adapter. You have to get instance from getInstance() method.

File: DataAccess/DataAccess.SQLite/LiteTable.hpp [hunk 3]
```diff
@@ -1,34 +1,36 @@
...
     }
 
-    LiteTable(const std::string& tableName, const std::string_view& options)
-        : QueryBuilder(DatabaseType::DB_LITE, tableName, options) 
+    /// Table constructor with options
+    LiteTable(const std::string& tableName, const std::string_view& options) : QueryBuilder(Utility::DatabaseType::DB_LITE, tableName)
     {
-        _adapter = LiteAdapter::getInstance<LiteAdapter>(options);
+        _adapter = LiteAdapter::Instance(options);
```
Comment (vDrWool) [inline DataAccess/DataAccess.SQLite/LiteTable.hpp:33]: Here too.

**Review by vDrWool — CHANGES_REQUESTED** [review]

File: DataAccess/DataAccess.Postgre/PostgreRepository.hpp [hunk 4]
```diff
@@ -23,14 +23,14 @@ class PostgreRepository : public IRepository
 
     virtual ~PostgreRepository() = default;
 
-    std::vector<Network::ChannelInfo> getAllChannelsList() override final;
```
[the remark this answers is not in the document]
Comment (StasDash) [reply DataAccess/DataAccess.Postgre/PostgreRepository.hpp:26]: final is same thing as override but with no futher override

File: DataAccess/DataAccess.SQLite/LiteTable.hpp (hunk 2 above)
Comment (StasDash) [reply DataAccess/DataAccess.SQLite/LiteTable.hpp:27]: okay, but `LiteAdapter::getInstance<LiteAdapter>();`{.cpp} doesn't look good

File: DataAccess/DataAccess.SQLite/LiteTable.hpp (hunk 3 above)
Comment (StasDash) [reply DataAccess/DataAccess.SQLite/LiteTable.hpp:33]: okay, but `LiteAdapter::getInstance<LiteAdapter>();` doesn't look good

**Review by vDrWool — APPROVED** [review]

**Review by vDrWool — APPROVED** [review]

**Review by Svidunovich — APPROVED** [review]

**Pull request merged**